ext/pcre/php_pcre.c

```diff
@@ -32,6 +32,7 @@
 #define PREG_PATTERN_ORDER			1
 #define PREG_SET_ORDER				2
 #define PREG_OFFSET_CAPTURE			(1<<8)
+#define PREG_NO_UTF8_CHECK			PCRE_NO_UTF8_CHECK
```

Comment:
I couldn't grasp the reasoning behind the various existing `PREG_*` constants, so I just used PCRE's value.
Please tell me if this should be changed.